### ISSUE TITLE
No longer getting warnings about names

### ISSUE THREAD

**Issue by alexaandru** [golang/lint#180]
Given this source:

```
package main

// Bar should not be OK as the first word...
const Foo = 42
```

I would expect golint to complain like: `comment on exported const Foo should be of the form "Foo ..."` but it does not, it seems happy with it.

Updated to the very latest golint right now, still does the same. Go version is ```go version go1.4 linux/amd64```.

Thank you :)

**Comment by dsymonds** [comment]
This sounds like #166. Maybe I did something wrong in 32a8716.

**Comment by alexaandru** [comment]
Sorry for opening a duplicate issue then, yes, it seems like that exact issue. I only searched in the Open issues before creating this one. Please feel free to close/merge. Thank you! :)

**Comment by dsymonds** [comment]
I can't reproduce your bug report. I do get golint reporting that. I suspect it's because you're on an old version of Go, and I'm using Go 1.5.

**Comment by alexaandru** [comment]
Thank you, upgraded to 1.5.2 and indeed, it works as expected. Cheers!
